quickstart/config/kratos/email-password/.kratos.yml

```diff
@@ -0,0 +1,72 @@
+selfservice:
```

Comment:
move to it's own subdirectory (`oidc-provider`) here https://github.com/ory/kratos/tree/master/contrib

Reply:
Removed, see https://github.com/ory/kratos/pull/538